maro/rl/policy/policy.py

```diff
@@ -60,6 +62,19 @@ def get_rollout_info(self):
     def get_batch_loss(self, batch: dict, explicit_grad: bool = False):
         pass
 
+    def data_parallel(self, *args, **kwargs):
```

Comment:
Maybe we should provide this interface for RL policies only. Rule-based policies don't need this, right?

Reply:
Exactly.

Reply:
Will it be moved to a sub-class of AbsPolicy?